picovico/cli/driver.py

```diff
@@ -3,15 +3,15 @@
 
 import six
 
-from . import decorators as cli_dec
-from . import log as cli_logger
-from . import profile_utils
-from . import prompt
-from . import project_driver as proj_driver
-from . import utils as pv_utility
-from .. import PicovicoAPI
-from ..components.base import PicovicoBaseComponent
-from .. import exceptions as pv_api_exceptions
+from picovico.cli import decorators as cli_dec
+from picovico.cli import log as cli_logger
+from picovico.cli import profile_utils
+from picovico.cli import prompt
+from picovico.cli import project_driver as proj_driver
```

Comment:
project_driver seems ok to me, but if you want to alias, then lets use pv_project_driver